docs/v2/api/filing-api.md

```diff
@@ -41,6 +41,98 @@ Quick links:
 - [`/institutions/<institution>/filings/<period>/submissions/<submissionId>/sign`](#signature)
 - [`/institutions/<institution>/filings/<period>/submissions/<submissionId>/summary`](#summary)
 
+
+### Institutions by id
+`/institutions/<lei>`
+
+`GET` - Returns the institution by its `lei` and its filings
+
+Example response:
+
+```json
+{
+    "institution": {
+        "activityYear": 2014,
+        "LEI": "10Bx939c5543TqA1144M",
```

Comment:
Is this `LEI` or `lei`? For the mock endpoint (and UI code changes) we were using lowercase `lei`.

Reply:
It is uppercase `LEI`.

Reply:
Would it be a problem to change it to lowercase?

Reply:
Sure, we can make it lowercase. I'll send another PR to this. Just need to make sure the refactoring doesn't affect the test cases.

Reply:
Cool with me, 👍.